src/Main.cpp

```diff
@@ -195,17 +195,54 @@ class SPIMessageSender : public AbstractTask<5 * configMINIMAL_STACK_SIZE> {
     ILogger& m_logger;
 
     void task() override {
+
         auto& spi = BSPContainer::getSpiEsp();
         while (true) {
-            if (!spi.isBusy()) {
-                const char message[] = "Sending message to ESP from task";
-                spi.send((uint8_t*)message, sizeof(message));
+            if (spi.isConnected()) {
+
+                HiveMindHostSerializer serializer(spi);
+                MessageSender messageSender(MessageHandlerContainer::getNetworkMessageQueue(),
```

Comment:
Use remotequeue, it was made for the ESP though the uart uses for the demo, I guess we could remove the uart task for now.